### PR TITLE
fix(docs): Update copyright year to 2024

### PR DESCRIPTION
**Pull request by imagoiq**
I didn't find a simple way to generate it or inject through Vite, and adding js seems cumbersome

### CODE REVIEW

File: .changeset/gorgeous-ties-help.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+'@swisspost/design-system-demo': patch
+'@swisspost/design-system-documentation': patch
+---
+
+Update copyright year to 2024 in the footer of all documentation.
```
Comment (imagoiq) [inline .changeset/gorgeous-ties-help.md:6]:
```suggestion
Updated copyright year to 2024 in the footer of all documentation.
```

File: packages/documentation/.storybook/blocks/footer.tsx [hunk 2]
```diff
@@ -83,7 +83,7 @@ export default (params: { pathToStoryFile?: String }) => (
       </div>
 
       <div className="container d-flex justify-content-between py-regular">
-        <strong>&copy; 2022 Swiss Post Ltd.</strong>
+        <strong>&copy; 2024 Swiss Post Ltd.</strong>
```
Comment (oliverschuerch) [inline packages/documentation/.storybook/blocks/footer.tsx:86]:
```suggestion
        <strong>&copy; {new Date().getFullYear()} Swiss Post Ltd.</strong>
```

Seems even more cumbersome to do this every year by hand 😄!

File: packages/demo/src/app/common/footer/footer.component.html [hunk 3]
```diff
@@ -30,7 +30,7 @@ <h2 class="bold">Support</h2>
 </footer>
 
 <footer class="container d-flex justify-content-between py-regular">
-  <span class="bold">&copy; 2022 Swiss Post Ltd.</span>
+  <span class="bold">&copy; 2024 Swiss Post Ltd.</span>
```
Comment (oliverschuerch) [inline packages/demo/src/app/common/footer/footer.component.html:33]: I leave this, as this is hopefully the last time we're doing it 🤞🏼

**Review by oliverschuerch — CHANGES_REQUESTED** [review]

**Review by oliverschuerch — APPROVED** [review]

File: packages/documentation/.storybook/blocks/footer.tsx (hunk 2 above)
Comment (imagoiq) [reply packages/documentation/.storybook/blocks/footer.tsx]: Forget that we are working with tsx and mdx for the doc (markdown is in my head). Good!

**Pull request merged**